contents/docs/product-analytics/stickiness.mdx

```diff
@@ -65,11 +65,9 @@ You can then create a [cohort](/docs/data/cohorts) of those users by clicking th
 
 ## Stickiness vs retention
```

Comment:
If this is the same between the two, we could use a snippet here?

Reply:
It is. Doubt it's worth the bother. I doubt we'll need to change this again any time soon.